frontend/ui/src/components/projects/components/project-card.tsx

```
@@ -133,7 +133,15 @@ export function ProjectCard({
           gap={"1"}
           className="rounded-md bg-gray-3 p-2 font-medium text-gray-11"
         >
-          <Clock className="h-4 w-4" />
+// At the top of the file, alongside other import statements
+import { iconSize } from "../../icons/icon"
+
+// ... other existing imports and code
```

Biome (1.9.4)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)